test-utils/runtime-tester/fuzz/README.md

```diff
@@ -0,0 +1,59 @@
+# Runtime Fuzz
+
+Currently only one target is present -- runtime-fuzzer.  
+This target will create random scenarios using Arbitrary trait
+and execute them. This will keep happening, until one scenario fails.  
+
+
+To run fuzz test:
+```bash
+rustup run nightly cargo fuzz run runtime-fuzzer
```

Comment:
I usually run fuzzing as

```suggestion
RUSTC_BOOTSTRAP=1 cargo fuzz run runtime-fuzzer
```

RUSTC_BOOTSTRAP is a cheat code that unlocks nightly features on the stable compiler. It shouldn't be used for anything which requires stability, but its fine for fuzzing I think.